src/App.tsx

```diff
@@ -1,61 +1,125 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+
+const currentTodos: Todo[] = todosFromServer.map(todo => {
+  const newProperty = usersFromServer.find(user => user.id === todo.userId);
+
+  return {
+    ...todo,
+    email: newProperty?.email || '',
+    name: newProperty?.name || '',
+  };
+});
 
 export const App = () => {
+  const [titleError, setTitleError] = useState(false);
+  const [selectError, setSelectError] = useState(false);
+
+  const [title, setTitle] = useState('');
+  const [select, setSelect] = useState(0);
+
+  const [todos, setTodos] = useState<Todo[]>(currentTodos);
+
+  const addTodo = () => {
+    const selectedUser = usersFromServer.find(user => user.id === select);
+
+    if (!selectedUser) {
+      return;
+    }
```

Comment:
Instead of silently returning when no user is selected, consider providing feedback to the user, such as setting an error state or displaying a message.